### ISSUE TITLE
[BUG] qwen3 not being finetuned on colab

### ISSUE THREAD

**Issue by swagdowdlez** [huggingface/autotrain-advanced#924]
### Prerequisites

- [x] I have read the [documentation](https://hf.co/docs/autotrain).
- [x] I have checked other issues for similar problems.

### Backend

Colab

### Interface Used

CLI

### UI Screenshots & Parameters

<img width="1561" height="255" alt="Image" src="https://github.com/user-attachments/assets/4a1d4ade-20e9-4134-85e0-5c8b5996f3dc" />

### Error Logs

Saving the dataset (1/1 shards): 100%
 43/43 [00:00<00:00, 3238.73 examples/s]
Saving the dataset (1/1 shards): 100%
 43/43 [00:00<00:00, 3596.46 examples/s]
INFO     | 2025-08-12 10:01:32 | autotrain.backends.local:create:20 - Starting local training...
INFO     | 2025-08-12 10:01:32 | autotrain.commands:launch_command:514 - ['accelerate', 'launch', '--num_machines', '1', '--num_processes', '1', '--mixed_precision', 'bf16', '-m', 'autotrain.trainers.clm', '--training_config', 'autotrain-llama32-1b-finetune/training_params.json', '--mixed_precision', 'bf16', '-m', 'autotrain.trainers.clm', '--training_config', 'autotrain-llama32-1b-finetune/training_params.json', '--mixed_precision', 'bf16', '-m', 'autotrain.trainers.clm', '--training_config', 'autotrain-llama32-1b-finetune/training_params.json', '--mixed_precision', 'bf16', '-m', 'autotrain.trainers.clm', '--training_config', 'autotrain-llama32-1b-finetune/training_params.json', '--mixed_precision', 'bf16', '-m', 'autotrain.trainers.clm', '--training_config', 'autotrain-llama32-1b-finetune/training_params.json', '--mixed_precision', 'bf16', '-m', 'autotrain.trainers.clm', '--training_config', 'autotrain-llama32-1b-finetune/training_params.json', '--mixed_precision', 'bf16', '-m', 'autotrain.trainers.clm', '--training_config', 'autotrain-llama32-1b-finetune/training_params.json', '--mixed_precision', 'bf16', '-m', 'autotrain.trainers.clm', '--training_config', 'autotrain-llama32-1b-finetune/training_params.json']
INFO     | 2025-08-12 10:01:32 | autotrain.commands:launch_command:515 - {'model': 'Qwen/Qwen3-235B-A22B-Instruct-2507', 'project_name': 'autotrain-llama32-1b-finetune', 'data_path': 'autotrain-llama32-1b-finetune/autotrain-data', 'train_split': 'train', 'valid_split': None, 'add_eos_token': True, 'block_size': -1, 'model_max_length': 2048, 'padding': 'right', 'trainer': 'sft', 'use_flash_attention_2': False, 'log': 'tensorboard', 'disable_gradient_checkpointing': False, 'logging_steps': -1, 'eval_strategy': 'epoch', 'save_total_limit': 1, 'auto_find_batch_size': False, 'mixed_precision': 'bf16', 'lr': 1e-05, 'epochs': 3, 'batch_size': 1, 'warmup_ratio': 0.1, 'gradient_accumulation': 8, 'optimizer': 'paged_adamw_8bit', 'scheduler': 'cosine', 'weight_decay': 0.0, 'max_grad_norm': 1.0, 'seed': 42, 'chat_template': None, 'quantization': 'int4', 'target_modules': 'all-linear', 'merge_adapter': True, 'peft': True, 'lora_r': 16, 'lora_alpha': 32, 'lora_dropout': 0.05, 'model_ref': None, 'dpo_beta': 0.1, 'max_prompt_length': 128, 'max_completion_length': None, 'prompt_text_column': 'autotrain_prompt', 'text_column': 'autotrain_text', 'rejected_text_column': 'autotrain_rejected_text', 'push_to_hub': True, 'username': 'swagdowdle', 'token': '*****', 'unsloth': False, 'distributed_backend': None}
11814
